assets/css/styles.css

```diff
@@ -0,0 +1,11 @@
+* {
+    font-family: sans-serif;
+}
+
+html {
+    font-size: 14px;
+    background-color: blue;
+    margin: 0;
```

Comment:
margin take 2px

Reply:
As Per standard UI, margin should be 0 for root element